vivarium/core/experiment.py

```diff
@@ -418,16 +418,17 @@ def get_path(self, path):
         '''
 
         if path:
-            step = path[0]
-            if step == '..':
+            first_step = path[0]
```

Comment:
I think `step` is still the appropriate name here, since the function is recursive this represents each step along the path eventually.

Reply:
```step``` was shadowing the ```ipdb``` function called ```step```.

Reply:
You can always access shadowed ipdb methods using !, like `!step`.

Reply:
good to know!